Docs/credits/Index.md

```diff
@@ -0,0 +1,26 @@
+### Owner
+
+- **Phenax:** [GitHub](https://github.com/Phenax), [Discord](https://discordapp.com/users/730358289849778186)
+
+### Maintiners
+- **MaskDuck:** [GitHub](https://github.com/MaskDuck), [Discord](https://discordapp.com/users/716134528409665586)
+
+- **William:** [GitHub](https://github.com/WilliamDavidHarrison),[Discord](https://discordapp.com/users/853158265466257448)
+
+- **Andrew:** [GitHub](https://github.com/andrewstech),[Discord](https://discordapp.com/users/598245488977903688)
+
+- **Mahir/Molai** [GitHub](https://github.com/mtgsquad),[Discord](https://discordapp.com/users/763767239018938368)
+
+- **TheBotlyNoob:** [GitHub](https://github.com/TheBotlyNoob),[Discord](https://discordapp.com/users/488802888928329753)
+
+- **Hackermon:** [GitHub](https://github.com/hackermondev),[Discord](https://discordapp.com/users/1044657759066525777)
+
+## Helpers
+
+- **Vaibhav:** [GitHub](https://github.com/VaibhavSys),[Discord](https://discordapp.com/users/914452175839723550)
+
+### Mods
```

Comment:
```suggestion
```

Reply:
I am a mod that's why there is a mod section

Reply:
@TerraPlayz That's for the Discord server